src/ebay-tourtip/ebay-tourtip-heading.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { ComponentProps, FC } from 'react'
+import classNames from 'classnames'
+
+type Props = ComponentProps<'h2'> & {
+    type?: string
+}
+
+const EbayTooltipHeading: FC<Props> = ({
+    className,
+    type = 'infotip',
```

Comment:
type is `tourtip` and should not be passed as a prop it should always be `tourtip__heading`